### PR TITLE
Add monthly build

### PR DESCRIPTION
**Pull request by phil-allen-msft**
Verifies the ability to continuously build.

### CODE REVIEW

File: azure-pipelines/official.yml [hunk 1]
```diff
@@ -4,6 +4,18 @@ trigger:
   paths:
     exclude: ["*.md"]
 
+schedules:
+  - cron: "0 8 23-29 * 0"
+    displayName: "Monthly build check"
+    branches:
+      include: 
+        - main
+        - rel/*
```
Comment (jimmylewis) [inline azure-pipelines/official.yml:13]: `rel/*` is a bad idea as it exists today.  We didn't use rel branches regularly, and those that do exist are super outdated in terms of build infrastructure so the builds will almost certainly fail.

I would stick with just main until another branching strategy becomes common.

File: azure-pipelines/official.yml [hunk 2]
```diff
@@ -4,6 +4,18 @@ trigger:
...
+      exclude: 
+        - ""
```
Comment (jimmylewis) [inline azure-pipelines/official.yml:15]: What does this accomplish?

File: azure-pipelines/official.yml [hunk 3]
```diff
@@ -4,6 +4,18 @@ trigger:
   paths:
     exclude: ["*.md"]
 
+schedules:
+  - cron: "0 8 23-29 * 0"
```
Comment (jimmylewis) [inline azure-pipelines/official.yml:8]: nit: add a comment summarizing the cron syntax for readability so it doesn't require a trip to the docs each time 😉

Comment (phil-allen-msft) [reply azure-pipelines/official.yml]: Added, and tweaked the dates so it is "4th Sunday" consistently

File: azure-pipelines/official.yml (hunk 2 above)
Comment (phil-allen-msft) [reply azure-pipelines/official.yml:15]: Denotes the location for the exclude, should we ever need one.  Will remove

**Review by jimmylewis — APPROVED** [review]

**Pull request merged**